tests/boardfarm_plugins/boardfarm_prplmesh/devices/prplmesh_prplwrt.py

```diff
@@ -126,14 +126,13 @@ def set_iface_to_bridge(self, iface: str) -> bool:
         self.sendline(ip_command)
         self.expect(self.prompt, timeout=10)
 
-    def set_iface_ip(self, iface: str, ip: IPv4Network) -> bool:
+    def set_iface_ip(self, iface: str, ip: IPv4Address, prefixlen :int) -> bool:
```

Comment:
Ok this fixup makes my previous comment about IPv4Network no longer relevant.
I still think we should use the existing IP if possible.

Reply:
Already done! Resolving.